client/www/index.html

```diff
@@ -86,6 +87,7 @@
 
 <!-- moment -->
 <script src="./scripts/vendor/moment.js"></script>
+<script src="./scripts/vendor/angular-moment.js"></script>
 
```

Comment:
do we need both 'moment' libraries or can we consolidate on the angular version?